en/components/input_group.md

```diff
@@ -0,0 +1,201 @@
+    <input igxInput name="fullName" type="text" required="required" />
+    <label igxLabel for="fullName">Full Name</label>
+</igx-input-group>
+```
+
+In the demonstration of the code below you can see we receive an asterisk next to the label as well as the input turns green/red when we write in it depending on whether the validation passes/fails.
+
+<div class="sample-container" style="height:100px">
+<iframe id="input-group-sample-2-frame" src='{environment:demosBaseUrl}/input-group-sample-2' width="100%" height="100%" seamless frameBorder="0"></iframe>
+</div>
+<div class="divider--half"></div>
+
+#### Data Binding
+Ignite UI for Angular Input supports one-way and two-way data-binding. Let's add a two-way data-binding to our input. Here is the code in our component:
+
+```typescript
+public user = {
+    fullName: ""
+};
+
+```
+
+and in our markup:
+
+```html
+<igx-input-group>
+    <input igxInput name="fullName" type="text" [(ngModel)]="user.fullName" required="required" />
+    <label igxLabel for="fullName">Full Name</label>
+</igx-input-group>
+```
+
+### Prefix & Suffix
+If you want to have an input prefix or suffix, you can use Ignite UI for Angular Prefix or Suffix. Let's add a new input field with prefix and suffix:
+
+```html
+<igx-input-group>
+    <igx-prefix>+359</igx-prefix>
+    <label igxLabel for="phone">Phone</label>
+    <input igxInput name="phone" type="text" [(ngModel)]="user.phone" />
+    <igx-suffix>
+        <igx-icon name="phone"></igx-icon>
+    </igx-suffix>
+</igx-input-group>
+```
+
+Here is how the sample looks:
+<div class="sample-container" style="height:100px">
+<iframe id="input-group-sample-3-frame" src='{environment:demosBaseUrl}/input-group-sample-3' width="100%" height="100%" seamless frameBorder="0"></iframe>
+</div>
+<div class="divider--half"></div>
+
+### Hints
+Ignite UI for Angular Hint provides a helper text placed below the input. The hint can be placed at the start or at the end of the input. The position of the `igxHint` can be set using the `position` property. Let's add a hint to our phone input:
+
+```html
+<igx-input-group>
+    <igx-prefix>+359</igx-prefix>
+    <label igxLabel for="phone">Phone</label>
+    <input igxInput name="phone" type="text" [(ngModel)]="user.phone" />
+    <igx-suffix>
+        <igx-icon name="phone"></igx-icon>
+    </igx-suffix>
+    <igx-hint position="start">Ex.: +359 888 123 456</igx-hint>
+</igx-input-group>
+```
+
+This is how the phone field with hint looks like:
+<div class="sample-container" style="height:110px">
+<iframe id="input-group-sample-4-frame" src='{environment:demosBaseUrl}/input-group-sample-4' width="100%" height="100%" seamless frameBorder="0"></iframe>
+</div>
+<div class="divider--half"></div>
+
+### Styling
+Our inputs could be styled differently by using the `type` property of the `igxInputGroup` component. Currently we support four different ways of styling: line (the default one), box, border and search. this is how they look like:
```

Comment:
Capitalize "This" if it's the start of a sentence.  Same here with "like".  If you want to use "like", you can write "This is what they look like".